modules/ROOT/content-nav.adoc

```diff
@@ -20,6 +17,10 @@
 *** xref:installation/kubernetes/helm-charts.adoc[]
 ** xref:installation/self-signed-certificate.adoc[]
 
+// * Upgrading
+* xref:upgrade/index.adoc[]
+** xref:upgrade/server.adoc[]
+** xref:upgrade/agent.adoc[]
```

Comment:
Good idea to move this after installation but maybe it should be after ' Adding managed instances' too as that section is about installing agents?